src/main/DAL/MetaQuestion/MetaQuestionRepository.js

```diff
@@ -97,10 +97,11 @@ class MetaQuestionRepository {
      * @param keywords Array of keywords: string[]
      * @returns The updated question.
      */
-    async addKeywordsToQuestion(qId, keywords) {
+    async setKeywordsToQuestion(qId, keywords) {
         const metaQuestion = await this.getMetaQuestion(qId);
         const dbKeywords = await this.#getDbKeywords(keywords)
-        await metaQuestion.addKeywords(dbKeywords);
+        await metaQuestion.setKeywords(dbKeywords);
+        await metaQuestion.save();
         await metaQuestion.reload();
```

Comment:
save = update DB from instance
reload = update instance from DB

Reply:
Yes but why do you reload ?
The database should not give you any new information, am I wrong ?
If I am correct, then you are making a bd request for no reason

Reply:
`setKeywords` sets the association between the keyword and the MQ. 
`save` writes these new association to the DB
`reload` updated the instance, this is needed because the instance carries with it the keywords (which are outside instances). I'm not sure `setKeywords` updates the instance in that manner, but I can check